src/components/Showdata.tsx

```diff
@@ -20,7 +20,7 @@ function Showdata({ handlestate, fik, tooglestate }: InputProps) {
     handlestate();
   };
```

Comment:
For this file and all other file move types and interfaces to a separate file name types.ts or intefaces.ts and then import those types and interfaces to respective files